src/echobot.c

```diff
@@ -208,8 +208,16 @@ void friend_message(Tox *tox, uint32_t friend_number, TOX_MESSAGE_TYPE type, con
 
 		const char *info_msg = "If you're experiencing issues, contact Impyy in #tox at freenode";
 		tox_friend_send_message(tox, friend_number, TOX_MESSAGE_TYPE_NORMAL, (uint8_t *)info_msg, strlen(info_msg), NULL);
-	} else if (strcmp("!callme", dest_msg) == 0) {
+	} else if (!strcmp("!callme", dest_msg)) {
 		toxav_call(g_toxAV, friend_number, audio_bitrate, 0, NULL);
+	} else if (!strcmp ("!videocallme", dest_msg)) {
+		toxav_call (g_toxAV, friend_number, audio_bitrate, video_bitrate, NULL);
+	} else {
+		char answer[TOX_MAX_MESSAGE_LENGTH];
```

Comment:
Sorry for being nitpicky. It should be ```TOX_MAX_MESSAGE_LENGTH + 1``` because snprintf also writes a null character and we can omit that when passing the message to toxcore. This would give us an extra character, hurray!